### ISSUE TITLE
Feature Request: Add an Ability To Add Cudtom JMS Connection.

### ISSUE THREAD

**Issue by iAweX** [jmstoolbox/jmstoolbox#61]
Please add an ability to add custom JMS connections for Artemis to use another protocols. For example using qpid-jms jars (for amqp connection) and jndi.

**Comment by tobiasoort** [comment]
Yep, i'm currently debugging a third party JMS connector JAR and would like to load this in a tool like JMSToolbox. That would be really great.
